### ISSUE TITLE
Font uploads and other mime types

### ISSUE THREAD

**Issue by d-bo** [PendalF89/yii2-filemanager#50]
Hello !
Just uploaded .ttf font but it has not appeared in the gallery.
Is it possible to upload fonts, pdfs, other files ?

**Comment by invaderhd** [comment]
Yes it is. but someone do a pull request that remove it from appearing. I already ask to revert that commit but I think I need to change it myself and do a pull request to solve the problem

**Comment by d-bo** [comment]
This would be a great feature. Not a just image file manager.
